04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -437,22 +450,22 @@ GDALinfo("NEON_RemoteSensing/HARV/HARV_RGB_Ortho.tif")
 
 ```
 
-> Bonus: So far we've created a RasterStack from multiple bands in one GeoTIFF, 
-> however, you can also create a RasterStack by combining single bands from 
-> multiple files. We will cover this in [Lesson 05 - Raster Time Series Data in R]({{ site.baseurl }}R/Raster-Times-Series-Data-In-R/).  
+Bonus: So far we've created a RasterStack from multiple bands in one GeoTIFF, 
+however, we can also create a RasterStack by combining single bands from 
+multiple files. We will cover this in [Lesson 05 - Raster Time Series Data in R]({{ site.baseurl }}/R/Raster-Times-Series-Data-In-R/).  
 
 ##RasterStack vs RasterBrick in R
 
 Both `RasterStack` and `RasterBrick` are `R` objects with multiple bands. The bands 
-in a `RasterStack` are stored anywhere on your computer. A 
+in a `RasterStack` are stored anywhere on the computer. A 
```

Comment:
The bands in a `RasterStack` are links to raster data that is located somewhere on your computer.